satpy/resample.py

```diff
@@ -109,6 +109,21 @@
     >>> my_area = AreaDefinition(...)
     >>> local_scene = scn.resample(my_area)
 
+Resize area definition in pixels
+--------------------------------
+
+Sometimes you may want to create a small image with fixed size in pixels.
+For example, to create an image of (y, x) pixels :
+
+    >>> small_scn = scn.resample(scn.finest_area().copy(height=y, width=x), resampler="nearest")
+
+
+.. warning::
+
+    Resizing with the `nearest` resampler will accept all sizes.
+    Be aware that resizing with native resampling only works if size is a whole factor of the original size.
```

Comment:
I'm torn on if the `nearest` sentence is necessary. It isn't just "nearest", but all non-native resamplers (now and in the future) that should work with all sizes. Actually now that I say that I'm not sure this example works if the input data is a SwathDefinition (ex. VIIRS data) instead of on a gridded area. I'm not sure the SwathDefinition has the copy method, but even if it does all resamplers might not take a swath definition as a target (ex. gradient search currently). Anyway, maybe it is clearer to be less detailed and only keep the second sentence. Thoughts?

Reply:
Oh ok, I'm not aware of all uses cases, that's seems better